src/__test__/helpers.ts

```diff
@@ -30,10 +31,18 @@ export function permutation(inputs: any[]): [any, any][] {
  * Create a primitive operand, evaluating as the passed value.
  * @param value 
  */
-export const operand = (value: any): Operand => new class {
-  type: EvaluableType
-  constructor(private readonly value: any) {
-    this.type = EvaluableType.Operand
-  }
+export const operand = (value: any): Operand => new class extends Operand {
+  type: EvaluableType = EvaluableType.Operand
+  constructor(private readonly value: any) { super() }
   evaluate() { return this.value }
+  simplify() { return this.value }
+  serialize(): Input { throw new Error('not implemented') }
 }(value)
+
+export const notSimplified = (): Operand => new class extends Operand {
```

Comment:
any concerns about just passing: simply: boolean into the operand, to avoid that secondary helper?
```
export const operand = (value: any, simply = false): Operand => new class extends Operand {}
```

Reply:
Yeah, we can do that.